### DISCUSSION TITLE
Add Stremio as streaming service option

### DISCUSSION THREAD

**Discussion by lobstercashtv** [javikr/sofatime#62] [Ideas]
Currently you can only access Stremio through “Related links”, which is a great feature, but it’d be better if you could set it as your default all encompassing streaming service. Then it would be much more convenient to find shows or movies through Sofa Time. Currently it’s pretty hidden.

**Comment by javikr** [comment]
Hi there,
Starting version 3.1.0, users can open in Stremio any episode or movie not only from the related links section but also from listings. "Open In Stremio" has been added also as a setting to enable or disable.
